pandas/io/excel.py

```diff
@@ -8,6 +8,7 @@
 import datetime
 import abc
 import numpy as np
+from collections import OrderedDict
```

Comment:
This should be imported from `pandas.compat` instead to ensure backwards compatibility with Python 2.6.